onediff_sd_webui_extensions/scripts/onediff.py

```diff
@@ -0,0 +1,203 @@
+import modules.scripts as scripts
+from modules import script_callbacks
+import modules.shared as shared
+from modules.processing import process_images
+
+import math
+import torch
+import oneflow as flow
+from einops import rearrange
+from oneflow import nn, einsum
+from sgm.modules.attention import default, CrossAttention
+from sgm.modules.diffusionmodules.util import GroupNorm32
+from omegaconf import OmegaConf, ListConfig
+from onediff.infer_compiler.transform.builtin_transform import torch2oflow
+
+
+@torch2oflow.register
+def _(mod, verbose=False) -> ListConfig:
+    converted_list = [torch2oflow(item, verbose) for item in mod]
+    return OmegaConf.create(converted_list)
+
+
+"""oneflow_compiled UNetModel"""
+_compiled = None
+
+
+# https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/master/modules/sd_hijack_optimizations.py#L142
+# https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/master/modules/sd_hijack_optimizations.py#L221
+class CrossAttentionOflow(nn.Module):
```

Comment:
```suggestion
class CrossAttention1f(nn.Module):
```

onediff 里目前都统一写成原类名 + `1f` 的方式（好像是xiaoyu 和 fengwen 约定的）

Reply:
这里参考了 [register_diffusers](https://github.com/Oneflow-Inc/onediff/blob/65a7147fe1cb2aca2576b6211bc3cf1ffee2fe44/src/infer_compiler_registry/register_diffusers/__init__.py#L7) 里的命名方式。

Reply:
> 这里参考了 [register_diffusers](https://github.com/Oneflow-Inc/onediff/blob/65a7147fe1cb2aca2576b6211bc3cf1ffee2fe44/src/infer_compiler_registry/register_diffusers/__init__.py#L7) 里的命名方式。

是的，后来都改成 Oflow 了